torch/testing/_internal/common_methods_invocations.py

```diff
@@ -52,6 +54,19 @@ def __init__(self, input, *, args=tuple(), kwargs=None):
         self.kwargs = kwargs if kwargs is not None else {}
 
 
+_NOTHING = object()  # Unique value to distinguish default anything else
```

Comment:
Add a comment describing this function and its intended use

Reply:
Comment added.